### ISSUE TITLE
No focus indicator on RGB sliders

### ISSUE THREAD

**Issue by patrickhlauke** [WebAccessibilityTools/CCA#8]
When navigating with the keyboard, the RGB sliders receive focus, but there's no visible focus indication.
